From f4040dea2e6e59e989ab6eac8af1e957de230928 Mon Sep 17 00:00:00 2001
From: Siarhei Yasinski <siarhei.yasinski@sintecs.eu>
Date: Tue, 27 Sep 2022 14:19:28 +0300
Subject: [PATCH] Panic strategy changed from abort to unwind.

Signed-off-by: Siarhei Yasinski <siarhei.yasinski@sintecs.eu>
---
 Cargo.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index c916bec..7eaaa49 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,7 +54,7 @@ members = [
 # Don't let panics from callbacks cross FFI boundary
 
 [profile.dev]
-panic = "abort"
+panic = "unwind"
 
 [profile.release]
-panic = "abort"
+panic = "unwind"
